apps/react-18-tests-v9/cypress-react-18.config.ts

```diff
@@ -1,30 +1,29 @@
 import * as path from 'path';
 import { baseConfig } from '@fluentui/scripts-cypress';
 
-// Exclude files that are not compatible with React 18 yet
-const excludedSpecs = [
-  '!' + path.resolve('../../packages/react-components/react-tag-picker/library/src/**/*.cy.{tsx,ts}'),
-  '!' + path.resolve('../../packages/react-components/react-tree/library/src/components/Tree/Tree.cy.tsx'),
-];
-
 // Include all tests from this app and the components package
-const includedSpecs = [
+const specs = [
   path.resolve('../../packages/react-components/**/library/src/**/*.cy.{tsx,ts}'),
   path.resolve('../../packages/react-components/react-tabster/src/**/*.cy.{tsx,ts}'),
 ];
-
-const specs = [...includedSpecs, ...excludedSpecs];
 const config = { ...baseConfig };
```

Comment:
Since we are now on latest Cypress, it's a good time to include those and make them fail. So we can fix it on follow-up PRs.